src/platform_impl/macos/view.rs

```diff
@@ -131,6 +131,15 @@ fn retrieve_keycode(event: &NSEvent) -> Option<VirtualKeyCode> {
     })
 }
 
+fn mouse_button(event: &NSEvent) -> MouseButton {
+    match event.buttonNumber() {
+        0 => MouseButton::Left,
+        1 => MouseButton::Right,
```

Comment:
Hmm, would these ever happen?

Reply:
Not with the NSOtherMouse... events, but the property is also set for the NSLeftMouse.../NSRightMouse... events. For the no-mouse-button event types, it's always 0, see https://developer.apple.com/documentation/appkit/nsevent/1527828-buttonnumber. Anyway, I think it's better to always convert the value here than to panic or return an Option.